inc/contenttemplates/templatemanager.class.php

```diff
@@ -176,4 +176,35 @@ public static function getSecurityPolicy(): SecurityPolicy {
       $functions = ['date', 'max', 'min','random', 'range'];
       return new SecurityPolicy($tags, $filters, $methods, $properties, $functions);
    }
+
+   /**
+    * Generate the documentation of the given parameters
+    *
+    * @param int $preset_parameters_key
+    *
+    * @return string
+    */
+   public static function generateMarkdownDocumentation(
+      int $preset_parameters_key
+   ): string {
+      $parameters = ParametersPreset::getByKey($preset_parameters_key);
+      $context = ParametersPreset::getContextByKey($preset_parameters_key);
+
+      $documentation = new TemplateDocumentation($context);
+      $documentation->addSection(__("Variables"), $parameters);
```

Comment:
All sections are containing variables, so using the `Root` keyword may help users.
```suggestion
      $documentation->addSection(__("Root variables"), $parameters);
```